Gauge.CSharp.Lib/ExecutionContext.cs

```diff
@@ -93,11 +93,13 @@ public class Specification {
         
         [Serializable()]
         public class StepDetails {
-            public StepDetails(String text, bool isFailing, string stackTrace, string errorMessage) {
+            public StepDetails(String text, bool isFailing, string stackTrace, string errorMessage, 
```

Comment:
Can we please preserve the old constructor and add an overload instead? This will help keep backward compatibility.

Reply:
Yes, that's reasonable
I've updated